### DISCUSSION TITLE
Add HTTP-Body as request example next to HTTP-HTTP/1.1

### DISCUSSION THREAD

**Discussion by christophdebaene** [scalar/scalar#902] [Ideas]
I am using [JSON-RPC](https://www.jsonrpc.org/specification) and successfully integrated with [Scalar](https://github.com/scalar/scalar).
It's always a `POST` request and I redirect the url always to `/jsonrpc` using the proxy. 
Below is an example of a `JSON-RPC` request

```json
{
  "id": "99",
  "jsonrpc": "2.0",
  "method": "Customers.GetCustomerById",
  "params": {
      "id": "12345"
   }
}
```

My suggestion for not having too much impact on `Scalar` for `JSON-RPC` in general, is to add `HTTP-Body` as request example next to `HTTP-HTTP/1.1`.  That would always show the request body in json format (and consequentially with syntax highlighting).

**Comment by hanspagel** [comment]
Can you share (part of) your OpenAPI file with us?

https://sandbox.scalar.com/

**Comment by christophdebaene** [comment]
Yes, I created a little sample https://sandbox.scalar.com/e/02H1x
Please let me know if you need more information.
